api/base.proto

```diff
@@ -30,3 +30,14 @@ message Node {
 message Endpoint {
   ResolvedAddress address = 1;
 }
+
+// Runtime derived uint32 with a default when not specified.
+message RuntimeUInt32 {
+  oneof value_specifier {
```

Comment:
I think this is not a oneof? We still need the default if the runtime_key does not exist.